commons/src/main/java/com/flipkart/flux/commons/dto/Eventsinfo.java

```diff
@@ -0,0 +1,31 @@
+package com.flipkart.flux.commons.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.Map;
+
+@NoArgsConstructor(access = AccessLevel.PACKAGE)
+@AllArgsConstructor
+public class Eventsinfo {
+
+    @Getter
+    @JsonProperty
+    private Map<String, EventData> eventData;
+
+    @NoArgsConstructor(access = AccessLevel.PACKAGE)
+    @AllArgsConstructor
+    @Getter
+    static class EventData {
```

Comment:
1. Can we make it public. Else how do we access
2. Is the data always String
3. Why isn't status an enum